evaluations-service-core/evaluations-service-models/src/main/java/ar/edu/itba/cep/evaluations_service/models/Exam.java

```diff
@@ -142,4 +216,67 @@ public String toString() {
                 "Duration: " + duration +
                 "]";
     }
+
+
+    // ================================
+    // Helpers
+    // ================================
+
+    /**
+     * Checks whether the exam can be modified.
+     *
+     * @throws IllegalEntityStateException If the exam cannot be modified.
+     */
+    private void verifyStateForUpdate() throws IllegalEntityStateException {
+        if (state != State.PENDING) {
+            throw new IllegalEntityStateException(PENDING_STATE_FOR_MODIFICATIONS);
+        }
+    }
+
+
+    // ================================
+    // Exam states
+    // ================================
+
+    /**
+     * An enum containing the different states in which an exam can be.
+     */
+    public enum State {
+        /**
+         * Indicates that an exam has not started yet.
+         */
+        PENDING,
```

Comment:
I'd change `PENDING` for `UPCOMING`. It sounds more clear, `PENDING` could be interpreted wrongly with something like `IN_PROGRESS`

Reply:
@juanmbellini